src/js/snapshots/markupSnapshot.spec.js

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (C) 2020 Knot.x Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import initStoryshots, { snapshotWithOptions } from '@storybook/addon-storyshots';
+
+initStoryshots({
+  test: snapshotWithOptions({
+    // eslint-disable-next-line consistent-return
```

Comment:
I wonder if we should tweak our eslint configuration? 🤔  There's a lot of `eslint-disable` comments in the code base and what's the point of eslint then? 
I guess one could argue it indicates the developer is aware of the possible issue